src/main/resources/db-scripts/clickhouse/derived_tables.sql

```diff
@@ -0,0 +1,46 @@
+-- clinical_data_derived
+DROP TABLE IF EXISTS clinical_data_derived
+CREATE TABLE IF NOT EXISTS clinical_data_derived
```

Comment:
Do we want to have this convention. Where we define the tables and the insert statements in one file? or the split file approach?